moveit_planners/chomp/chomp_motion_planner/include/chomp_motion_planner/chomp_planner.h

```diff
@@ -38,21 +38,21 @@
 #define _CHOMP_PLANNER_H_
 
 #include <chomp_motion_planner/chomp_parameters.h>
-#include <moveit_msgs/MotionPlanDetailedResponse.h>
-#include <moveit_msgs/MotionPlanRequest.h>
+#include <moveit/planning_interface/planning_request.h>
+#include <moveit/planning_interface/planning_response.h>
 #include <moveit/planning_scene/planning_scene.h>
 
 namespace chomp
 {
 class ChompPlanner
 {
 public:
-  ChompPlanner();
-  virtual ~ChompPlanner(){};
+  ChompPlanner() = default;
```

Comment:
Is this even worth keeping is we don't do anything in it?

Reply:
Otherwise, the default constructor wouldn't be created.

Reply:
I think it should. When you don't declare it at all, you get the default constructor. When you have one declared, the compiler won't generate the default one for you but the `default` keyword can be used to force generating it. Unfortunately I don't have a setup available ATM where I could double check it but I'm pretty certain.

The story about the destructor is slightly different as if you declare it virtual you are allowing derived classes to do a proper cleanup.

Reply:
I believe the default constructor is only created by default if there are no other constructors defined. But, I might be wrong. Nevertheless, it's not wrong to explicitly declare that we want to have the default constructor :wink:

Reply:
Yeah and as far as I see this is the only constructor being defined. This conversation is dragging for too long, let's go to merge :D